apps/vc-api/docs/openapi.json

```
@@ -649,6 +649,17 @@
         },
         "required": ["type"]
       },
+      "VpRequestPresentationDefinitionQueryDto": {
+        "type": "object",
+        "properties": {
+          "presentationDefinition": {
+            "type": "object",
+            "description": "An object conforming to the Presentation Definition specification"
```

Maybe put a link to the spec here?
```suggestion
            "description": "An object conforming to the Presentation Definition specification https://identity.foundation/presentation-exchange/#presentation-definition"
```

---

Sure. My bad when copy-pasting from the comment.